.azure-pipelines/stage/prechecks.yml

```diff
@@ -37,7 +37,7 @@ parameters:
 jobs:
 - job: prechecks
   displayName: Precheck
-  timeoutInMinutes: 20
+  timeoutInMinutes: 30
```

Comment:
updating the build-tools toolchains causes everything RBE to rebuild - proto_sync which now takes ~1 min for most types of change has to completely rebuild the api so can take slightly > 20mins